data/human/hails.txt

```diff
@@ -5824,6 +5827,279 @@ phrase "friendly deep"
 		"piracy"
 		"social regression"
 		"corruption"
+		"hostility and war"
+		"instability"
+		"anarchism"
+		"hostility"
+		"base values"
+	word
+		"."
+
+phrase "friendly deep"
+	word
+		"We"
+		"Our ships"
+		"Our security forces"
+		"Our fleets"
+		"Our forces"
+	word
+		" "
+	word
+		"" 3
+		"are called to "
+		"are diligent to "
+		"must constantly "
+		"diligently "
+		"always "
+		"must "
+		"shall "
+	word
+		"safeguard"
+		"defend"
+		"protect"
+		"guard"
+		"shield"
+		"advance"
+		"work in advancing"
+	word
+		" "
+	word
+		"" 3
+		"the Deep's "
+		"those who bring "
+		"the ones who serve "
+		"our shared "
+	word
+		""
+		"valued "
+		"invaluable "
+		"hard-fought "
+		"hard-gained "
+	word
+		"democracy"
+		"commonwealth"
+		"sovereignty"
+		"autonomy"
+		"prosperity"
+		"peace"
+		"economic prosperity"
+		"economic progress"
+		"progress"
+		"scientific progress"
+		"social cohesion"
+		"peace, prosperity, and progress"
+		"peace among the stars"
+	word
+		"."
```

Comment:
```suggestion
		"" 3
		"${deep adjectives}" 12
		"the Deep's "
		"the Deep's ${deep adjectives}" 4
		"those who bring "
		"those who bring ${deep adjectives}" 4
		"the ones who serve "
		"the ones who serve ${deep adjectives}" 4
		"our shared "
		"our shared, ${deep adjectives}" 4
	word
		"democracy"
		"commonwealth"
		"sovereignty"
		"autonomy"
		"prosperity"
		"peace"
		"economic prosperity"
		"economic progress"
		"progress"
		"scientific progress"
		"social cohesion"
		"peace, prosperity, and progress"
		"peace among the stars"
	word
		"."

phrase "deep adjectives"
	word
		"valued "
		"invaluable "
		"hard-fought "
		"hard-gained "
```
When you have something like "our shared" followed by an adjective like "invaluable" it becomes "our shared, invaluable" whereas when it is just followed by a word like "progress" it becomes "our shared progress." This allows the combinations you want while being, as far as I can tell, grammatically correct. I looked at the other combinations and after this there shouldn't be any problems as far as I can see.